client/src/components/SearchCombo.js

```diff
@@ -1,68 +1,79 @@
 // @flow
 import React from 'react';
+import { connect } from 'react-redux';
 import Dropdown from 'react-bootstrap/Dropdown';
 import FormControl from 'react-bootstrap/FormControl';
 
 import { BackendClient } from '../requests';
+import { updateSearchValue, doFetchPlaces } from '../actions';
 
 let connection = new BackendClient();
 
 type Props = {|
   style?: string,
-  className?: string
+  className?: string,
+  updateSearchValue: (string) => void,
+  doFetchPlaces: () => void
 |};
 
 type State = {|
   value: string,
   tags: ?[]
 |};
 
-export class SearchCombo extends React.Component<Props, State> {
+class SearchCombo extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = { value: '', tags: null };
```

Comment:
actually it can't because there are three states for `tags` here:
1. Non-empty array: will be displayed
2. Empty array: will show the message: "No results" or something similiar
3. null: will hide the result dropdown